src/components/Friends/SearchFriends.js

```diff
@@ -118,15 +160,39 @@ export const SearchFriends = () => {
         return <div key={x.id}>{entry}</div>;
       })
     ) : (
-      <></>
+      <CirclesBackground />
     );
 
   useEffect(() => {
     getPeople();
   }, [search]);
 
+  useEffect(() => {
+    if (statusChange) {
+      if (text === 'send friend request') {
```

Comment:
Checking strings like this is usually a bad practice because if the text changed the code will break which shouldn’t happen.
A solution would be to use enums. If you we want we can have a short call and see how

Reply:
Yeah, I was cutting corners a bit, I'm not going to lie.  I had to pass a lot of local state around and didn't want to create yet another local state so I made it dependent on the 'text.'  But you're probably right, creating a separate enum for it is probably what should happen.